src/parquet/reader.cc

```diff
@@ -106,9 +106,9 @@ ColumnReader* RowGroupReader::Column(size_t i) {
   source->Seek(col_start);
 
   // TODO(wesm): Law of demeter violation
-  size_t bytes_read = source->Read(input->size(), input->data());
-
-  if (bytes_read != input->size()) {
+  ScopedInMemoryInputStream* pInput = static_cast<ScopedInMemoryInputStream*>(input.get());
+  size_t bytes_read = source->Read(pInput->size(), pInput->data());
```

Comment:
@wesm I'll fix the variable name.
@nongli Using clang-format is helpful. Is it possible to check during a Travis build whether code confirm to style?